### PR TITLE
Testing the occurrence pipeline

### PR DESCRIPTION
**Pull request by peterdesmet**
Minor branch to test the occurrence pipeline. I currently get:

screen shot 2018-02-03 at 10 32 29

There doesn't seem to be a place in https://github.com/trias-project/pipeline/blob/test-occ-download/src/occurrences.Rmd where user/pass can be provided. Would be nice if this was asked to the user using:

```r
gbif_username <- rstudioapi::askForPassword("GBIF username")
gbif_password <- rstudioapi::askForPassword("GBIF password")
```

And then pass these on to `rgbif`.

### CODE REVIEW

**Comment by peterdesmet** [comment]
Please continue development on this feature in this branch.

File: src/occurrences.Rmd [hunk 1]
```diff
@@ -20,9 +21,15 @@ library(trias)
...
+eu_concern_species <- 
+  read_delim(file = input_checklist, 
+             "\t", escape_double = FALSE, 
+             col_types = cols(
+               entry_into_force = col_date(
+                 format = "%Y-%m-%d")), 
+             trim_ws = TRUE)
 taxonKeys <- eu_concern_species["backbone_taxonKey"]
-countries <- list(country = c("BE"))
+countries <- list(country = c("LU"))
```
Comment (stijnvanhoey) [inline src/occurrences.Rmd]: I'm just wondering, why always the `list` -> `unlist` combination?

Comment (damianooldoni) [reply src/occurrences.Rmd]: You are right. Solved in 1bd0b001252458058c157c77c55d407fd7217ca8

**Comment by peterdesmet** [comment]
- File renamed to `get_occurrences.Rmd`
- Documented/rewrote large sections of file
- Used `rstudioapi::askForPassword("GBIF username")` to request for credentials
- Works with run, not yet with knit (because of file paths and rstudio API)
- Ran for modelling species

Have a feature request coming up for function `update_download_list()`. 😄 

Accepting this PR.

**Pull request merged**